### DISCUSSION TITLE
No map update indicator anymore?

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#466] [Q&A] [answered]
I am using OAM maps and the freshly updated February maps are available and shown in the update folder. But there is no indicator on the map tile and announcement anymore that updates are available. Is it on purpose?

**Comment by devemux86** [comment, marked as answer]
@kds112 there were reports for wrong operation, e.g.:
- https://github.com/devemux86/cruiser/discussions/435

So updates are available in the 3rd tab of the download manager (as always).
We will review the operation of the indicator in the future.

**Comment by devemux86** [comment]
BRouter is updated daily and is important in routing.

Offline maps / poi have not such important updates to keep looking for them.

**Comment by WolfgangZ42** [comment]
As far as I remember there was a update counter in the first setting panel and in the map setting panel if map (or poi) updates are available:
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/52dc25eb-834f-412a-8714-fb34b9792603"/>    <img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/04746252-cd44-455e-9743-19f67195aeef"/>
Is that no longer implemented? Surely I can take a look and initiate then the updates:
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/765885c2-5edf-4e5b-9168-b0c188d8c6e6"/>

**Reply by devemux86** [reply]
It was disabled because some users reported that it did not work correctly.
I will review it and we could add it again in the future. :slightly_smiling_face:
